api/wd/index.md

```diff
@@ -87,38 +89,50 @@ e.g. `http://localhost:1234/some/route`.
 assert.match(/^https:/, browser.getUrl());
 ```
 
-#### `browser.getStatusCode()`
+### browser.getUrlObject(): Promise<URL>
+<span class="new-in">Added in: testium-driver-wd v3.1.0</span>
+
+Returns a [WHATWG URL instance](https://nodejs.org/dist/latest-v13.x/docs/api/url.html#url_the_whatwg_url_api) of the current url.
+
+### browser.getStatusCode(): Promise<number>
 
 Returns the response status code for the current page.
 
-#### `browser.assertStatusCode(statusCode)` (deprecated)
+### browser.loadPage(url: string[, options?: loadPageOpts]): Promise<void>
```

Comment:
no need for the `[...]` - that's what the `?` means

Reply:
thanks .. I oversaw this one